app/src/pages/profilePage/profile.module.css

```diff
@@ -0,0 +1,138 @@
+#profile {
```

Comment:
nice use of:
- grid
- flexbox
- pseudo classes
- scrollbarstyling
- modern font styles
styling looks like it is accessible to me